packages/postcss-convert-values/src/lib/convert.js

```diff
@@ -31,41 +31,32 @@ function dropLeadingZero(number) {
   return value;
 }
 
-function transform(number, unit, conversion) {
-  const lowerCasedUnit = unit.toLowerCase();
-  let one, base;
-  let convertionUnits = Object.keys(conversion).filter((u) => {
-    if (conversion[u] === 1) {
-      one = u;
-    }
-    return lowerCasedUnit !== u;
+function transform(number, originalUnit, conversions) {
+  let conversionUnits = [...conversions.keys()].filter((u) => {
+    return originalUnit !== u;
   });
 
-  if (lowerCasedUnit === one) {
```

Comment:
I don't think this `one` variable has any purpose any more.
if `lowerCasedUnit === one` then  `conversion[lowerCasedUnit] === 1` since `conversion[one] === 1`.
So `base / conversion[lowerCasedUnit] === base * conversion[lowerCasedUnit]`, correct?